docs/README.md

```diff
@@ -210,7 +219,7 @@ Now, you are able to use all the blocks exported by the `tab-layout` app. Check
 | `blockClass` | `string` | Block ID of your choosing to be used in [CSS customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization). | `undefined` |
```

Comment:
Atualizar esse link para o do Developer Portal

https://developers.vtex.com/docs/guides/vtex-io-documentation-using-css-handles-for-store-customization